editor/plugins/path_3d_editor_plugin.h

```diff
@@ -39,6 +39,18 @@
 
 class MenuButton;
 
+// Map handle id to control point id and handle type.
+enum HandleType {
+	IN,
+	OUT,
+	TILT,
+};
```

Comment:
```suggestion
// Map handle id to control point id and handle type.
enum HandleType {
	HANDLE_TYPE_IN,
	HANDLE_TYPE_OUT,
	HANDLE_TYPE_TILT,
};
```

The format of the enum must follow Godot's style.

FYI, if make it to:

```suggestion
// Map handle id to control point id and handle type.
enum HandleType {
	HANDLE_TYPE_IN,
	HANDLE_TYPE_OUT,
	HANDLE_TYPE_TILT,
	HANDLE_TYPE_MAX,
};
```

You can write some constant int value in line like:

```cpp
const int handle_idx = idx * int(HANDLE_TYPE_MAX) + int(HANDLE_TYPE_IN);
```

Well, as to the later, it is not a blocker.

Reply:
done.

> You can write some constant int value in line like:

thanks. With cache `_secondary_handles_info` it is no longer needed. So I think keep the code as is.